bio/varscan/somatic/wrapper.py

```
@@ -35,7 +35,7 @@
     mpileup = "--mpileup 1"
 elif all(pileup in snakemake.input.keys() for pileup in pileup_pair):
     # Case there are two separate pileup files
-    in_pileup = " {snakemake.input.normal_pileup}" " {snakemakeinput.tumor_pileup} "
+    in_pileup = f"{snakemake.input.normal_pileup} {snakemakeinput.tumor_pileup}"
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)